cli/viam/main.go

```diff
@@ -45,8 +45,9 @@ func main() {
 	var logger golog.Logger
 
 	app := &cli.App{
-		Name:  "viam",
-		Usage: "interact with your robots",
+		Name:            "viam",
+		Usage:           "interact with your Viam robots through the command line",
+		HideHelpCommand: true,
```

Comment:
I've hidden the `help` command for every command. I find it to be confusing/redundant with the `--help` command option that every command already has.

Reply:
Agree with this!

Q: does `HideHelpCommand: true` also suppress `help` appearing as a valid subcommand for invocations like so:

```
bananajr:~ andf$ viam auth --help | grep "one command"
   help, h             Shows a list of commands or help for one command
```

Reply:
`viam auth --help | grep "one command"` now returns nothing. `HideHelpCommand` does indeed remove the `help, h` subcommand from the usage text.

Reply:
Great! Thanks for confirming!